src/app/components/toast/index.html

```diff
@@ -0,0 +1,92 @@
+    <p>
+      The toast service launches a container over a page's content to display small messages. The <stache-code>SkyToastService</stache-code> provider launches toast messages and provides a common look-and-feel. The <stache-code>SkyToastMessage</stache-code> class watches for and triggers toast events. It also allows you to close messages.
+    </p>
+    <sky-alert
+      alertType="info">
+      The <stache-code>SkyToastService</stache-code> dynamically generates the toast component and appends it directly to the document's <stache-code>body</stache-code> element.
+    </sky-alert>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties sectionHeading="Flyout service methods">
+    <sky-demo-page-property
+      propertyName="open(config)">
+      Opens a message and displays the configured content. The <stache-code>config</stache-code> parameter is of type <stache-code>SkyToastConfig</stache-code>. This method returns the SkyToastMessage instance.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="openMesage(message, config?)">
+      Opens a message toast and displays the provided message. The <stache-code>message</stache-code> parameter is of type <stache-code>string</stache-code> and determines the message to be displayed. The optional <stache-code>config</stache-code> parameter is of type <stache-code>SkyToastConfig</stache-code> and adds additional configuration options to the toast. This method returns the SkyToastMessage instance.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="openTemplatedMessage(customComponentType, config?, providers?)">
+      Opens a custom toast and renders the provided component in a toast. The <stache-code>customComponentType</stache-code> parameter is of type <stache-code>Type&lt;SkyToastCustomComponent&gt;</SkyToastCustomComponent></stache-code> and determines the component to be rendered. The optional <stache-code>config</stache-code> parameter is of type <stache-code>SkyToastConfig</stache-code> and adds additional configuration options to the toast. The optional <stache-code>providers</stache-code> parameter is of type <stache-code>Provider[]</stache-code> and specifies an array of custom providers to pass to the <stache-code>customComponentType</stache-code>'s constructor. This method returns the SkyToastMessage instance.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="SkyToastConfig type">
+    <sky-demo-page-property
+      propertyName="message"
+      isOptional="true">
+      Sets the message that the toast displays. This may not be provided with the <stache-code>customComponentType</stache-code> property.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="customComponentType"
+      isOptional="true">
+      Sets the component that the toast renders. This may not be provided with the <stache-code>message</stache-code> property.
```

Comment:
Same question about the second sentence as in the previous note. Are we trying to explain that the two properties work in conjunction? If so, maybe we should change this entry to something like "Sets the toast component that displays the message from the `message` property." (And the previous first sentence would be the reverse: "Specifies the message to display in the toast component from the `customComponentType` property.")